src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -113,6 +117,16 @@ export default {
         addTelemetry(telemetryDomainObject) {
             this.telemetryObjs.push(telemetryDomainObject);
         },
+        removeTelemetry(telemetryDomainObjectIdentifier) {
+            let index = _.findIndex(this.telemetryObjs, (obj) => {
+                let objId = this.openmct.objects.makeKeyString(obj.identifier);
+                let id = this.openmct.objects.makeKeyString(telemetryDomainObjectIdentifier);
+                return objId === id;
+            });
+            if (index > -1) {
+                this.telemetryObjs.splice(index, 1);
+            }
+        },
         addCondition(event, isDefault) {
             let conditionDO = this.getConditionDomainObject(!!isDefault);
```

Comment:
DO in the variable name is not descriptive.

Reply:
DONE